docs/ScalarDLBackupCreationRestoration.md

```diff
@@ -0,0 +1,90 @@
+# Scalar DL Backup Creation on Kubernetes
+
+This guide shows you how to create and restore Scalar DL backups on Cosmos DB and DynamoDB without data inconsistencies.
+
+## Prerequisites
```

Comment:
I think this doc is overall redundant with the backup and restore guide.
If it is saying specifically for k8s, talk about k8s-specific things mainly here.

Reply:
> I think this doc is overall redundant with the backup and restore guide.

Yes, I admit that.
There are a lot of concept repetitions.
I will update the document without any concept repetitions.

Reply:
>If it is saying specifically for Scalar DL and k8s, talk about Scalar DL and k8s specific things here.

Ok, I will update the document with above details